app/controllers/applications/build_controller.rb

```diff
@@ -48,8 +53,8 @@ def show # rubocop:disable CyclomaticComplexity
   end
 
   def update
-    evidence_check_selection
-    create_payment_if_needed
+    # evidence_check_selection
+    # create_payment_if_needed
```

Comment:
Most of the controller code is not used any more, so I didn't bother cleaning it up. I'm hoping to remove it all together later today.